sound/soc/sof/Kconfig

```diff
@@ -40,12 +40,14 @@ config SND_SOC_SOF_ACPI_DEV
 config SND_SOC_SOF_OF
 	tristate "SOF OF enumeration support"
```

Comment:
@plbossart i think this code is more clear. On the other hand saying that snd-sof-of is a 'helper' doesn't make much sense in  my understanding.

snd-sof-of is the starting point for the OF platform driver:

```
sound/soc/sof/sof-of-dev.c

/* DT driver definition */
static struct platform_driver snd_sof_of_driver = {
»       .probe = sof_of_probe,
»       .remove = sof_of_remove, 
»       .driver = {
»       »       .name = "sof-audio-of",
»       »       .pm = &sof_of_pm,
»       »       .of_match_table = sof_of_ids,
»       },
};
module_platform_driver(snd_sof_of_driver);
```

Reply:
it's not a pure library indeed, but the point is that sof-of-dev is at the end of the 'select' clauses, it does not select anything on its own. Maybe 'common module' is a better wording.